### DISCUSSION TITLE
Does Renovate Support only using digests?

### DISCUSSION THREAD

**Discussion by pspurlock07** [renovatebot/renovate#15624] [Request Help]
Just want to make sure I am not understanding something incorrectly. In our helm charts we our working towards using only sha digests for our images as follows:

```
image:
  repo: gitlab.com
  tag: sha256:123456789
```
 It seems the only way I can get renovate to work is by doing this:
 
 ```
 image:
   repo: gitlab.com
   tag: v1.0.0@sha256:123456789
 ```
 
 Am I correct in understanding that example 1 is not currently supported by Renovate?

**Comment by viceice** [comment]
maybe, what about using latest tag in front of the digest?

if you provide a minimal reproduction repo, then we can probably extend renovate to support digest only.
